src/App.tsx

```diff
@@ -137,7 +129,7 @@ const App = (): JSX.Element => {
     (async () => {
       try {
         dispatch(isVaultClientLoaded(false));
-        const vault = await window.bridge.vaults.get(id, COLLATERAL_TOKEN_ID_LITERAL);
+        const vault = await window.bridge.vaults.get(id, RELAY_CHAIN_NATIVE_TOKEN);
```

Comment:
@peterslany 
I totally agree with you. I also noticed that but did not think of raising the potential issue. As we started supporting multi collaterals, this check has to be updated properly. But in this refactoring task, I focused on porting and tried to keep the scope. Thank you for your notice.

@peterslany cc @tomjeatt 
I think we should create a ticket because it could cause a critical issue. Let me create one.